### ISSUE TITLE
Menu

### ISSUE THREAD

**Issue by TURX** [aitc19x/VR-JC#7]
There are three pausible solutions to the menu so far.

**Solution 1 (3DoF)**

All the scenes are floating as spheres around the player. Submenus can be implemented by a destruct effect (separate the original sphere to serveral smaller spheres, and combine the smaller spheres back to the original one after losing the ray)

Pros: convenient
Cons: limited capacity (may be able to solve with submenus)

**Solution 2 (6DoF)**

All the scenes are on the ground as minified objects around the player.

Pros: no need to calculate a new position for user entered the scene (use the current position instead)
Cons: area consuming

**Solution 3 (3DoF)**

According to @may-oz's proposal of design ([ホーム画面アイデア改.pdf](https://github.com/aitc19x/VR-JC/files/4673062/default.pdf)), maps can be used, and the user can click the destination.

Pros: easiest to implement

Implement them to decide which to use.

If you have any more ideas, feel free to submit yours by reply.

**Comment by sei0o** [comment]
Cons of solution 1 can be solved making "submenu/child menu" appear after one was selected. I have never experienced 6DoF VR, that makes it difficult for me to compare them though.

**Comment by sei0o** [comment]
> Cons: Internet consuming, Google APIs blocked in China

Is it necessary to use Google APIs for the implementation for it? I mean, those maps in PDF doesn't have to be ones from Google Maps?  Plus, it seems difficult to "move pins" on VR apps.

I been searching how to create something similar to "white dot selector" in Google Expedition to select menu items or to show descriptions, say, of statues or places, but still don't get any ideas. Do you? I bet CloudAnchors or Anchors can do that, however, I'm so confused since we are using deprecated libraries to get 'em work across Android and iOS (>_<)

**Comment by TURX** [comment]
> Is it necessary to use Google APIs for the implementation for it? I mean, those maps in PDF doesn't have to be ones from Google Maps? Plus, it seems difficult to "move pins" on VR apps.

You are right. It is not necessary for us to use Google APIs to implement the map.

> I been searching how to create something similar to "white dot selector" in Google Expedition to select menu items or to show descriptions, say, of statues or places, but still don't get any ideas. Do you?

It is ok to just use "click" action instead of moving a pin to locate the destination as described in Ozaki-san's proposal.

> I'm so confused since we are using deprecated libraries to get 'em work across Android and iOS (>_<)

The GVR libraries are not deprecated when the project initiated (August 2019), and currently there is no good alternative of GVR that can integrate with Unity and smartphones. Since there are still many documents on the Internet so far, I think it is good to persist using these libraries.
